content/en/apps/guides/hosting/monitoring/postgres-ingest.md

```diff
@@ -0,0 +1,437 @@
+      failure_type
+    ORDER BY 
+      count DESC
+  metrics:
+    - failure:
+        usage: "LABEL"
+        description: "Name of the failure"
+    - sequence:
+        usage: "TABLE"
+        description: "Replication failure reasons"
+```
+
+Note that you can use any value on the first line, seen as  `replication_failure_reasons` above.  It will be the field name we'll use in the 4th step below when exploring the new data in Grafana.
+
+### Add new Postgres Exporter
+
+In a new file  `~/extra-sql-compose.yml`, define your new Postgres exporter as well as add a mount to the existing Grafana and Prometheus services. Note that the `DATA_SOURCE_NAME` value will need to have the following variables added to you `.env` file you created to deploy Watchdog:
+
+* `EXTRA_SQL_USER` - Postgres user to use when logging in
+* `EXTRA_SQL_PASS` - Password for `EXTRA_SQL_USER` above
+* `EXTRA_SQL_SERVER` - URL or IP for your Postgres server
+* `EXTRA_SQL_PORT` - Port of server, defaults to `5432` it not declared.
+* `EXTRA_SQL_DATABASE` - Actual string of database name (eg `extra_monitoring` or `health_stats`), will be different for each install.
+
+```yaml
+services:
+
+  prometheus:
+    volumes:
+      - ./extra-sql-prometheus.yml:/etc/prometheus/scrape_configs/extra-sql-prometheus.yml:ro
+
+  grafana:
+    volumes:
+      - ./extra-sql-dashboard.json:/etc/grafana/provisioning/dashboards/CHT/cht_admin_extra_sql.json:ro
+
+  extra_sql_exporter:
+    image: prometheuscommunity/postgres-exporter:latest
+    command:
+      # disables the collection of all metrics except for custom queries 
+      - '--no-collector.database'
+      - '--no-collector.postmaster'
+      - '--no-collector.process_idle'
+      - '--no-collector.replication'
+      - '--no-collector.replication_slot'
+      - '--no-collector.stat_bgwriter'
+      - '--no-collector.stat_database'
+      - '--no-collector.statio_user_tables'
+      - '--no-collector.stat_statements'
+      - '--no-collector.stat_user_tables'
+      - '--disable-default-metrics'
+      - '--disable-settings-metrics'
+    volumes:
+      - ./extra-sql-queries.yml:/extra-sql-queries.yml
+    environment:
+      DATA_SOURCE_NAME: "postgresql://${EXTRA_SQL_USER:-NO DB USER SPECIFIED}:${EXTRA_SQL_PASS:-NO DB PASSWORD SPECIFIED}@${EXTRA_SQL_SERVER:-.NO DB SERVER SPECIFIED}:${EXTRA_SQL_PORT:-5432}/${EXTRA_SQL_DATABASE:-.NO DB SPECIFIED}?sslmode=disable"
+      PG_EXPORTER_EXTEND_QUERY_PATH: "/extra-sql-queries.yml"
+    restart: always
+    networks:
+     - cht-watchdog-net
+```
+
+If there's a missing mount target, then Docker will error.  Avoid the error by creating an empty file called `~/extra-sql-dashboard.json`.  We'll use this in the last step below
+
+### Adding new scrape config
+
+Create the `~/extra-sql-prometheus.yml` file and point the config to our new Postgres Exporter which is `extra_sql_exporter:9187` on the last line below.  This will tell Prometheus to scrape the new data every 1 minute:
```

Comment:
```suggestion
Create the `~/scrape_config.custom-sql.yml` file and point the config to our new Postgres Exporter (`custom_sql_exporter:9187`).  This will tell Prometheus to scrape the new data every 1 minute:
```